libraries/bezier-rs/src/subpath/manipulators.rs

```diff
@@ -0,0 +1,147 @@
+use super::*;
+use crate::{consts::MAX_ABSOLUTE_DIFFERENCE, utils::f64_compare, ComputeType};
+
+impl Subpath {
+	/// Calculate the point on the subpath based on the parametric `t`-value provided.
```

Comment:
This sounds inaccurate (it doesn't just calculate a point). It inserts an anchor, and handles, at a certain point along the subpath where each constituent segment accounts for an equal proportion of the 0..1 interval.